Dockerfile

```diff
@@ -17,12 +17,12 @@ WORKDIR /tmp/grafana
 COPY package.json yarn.lock .yarnrc.yml ./
 COPY .yarn .yarn
 COPY packages packages
+COPY public public
 COPY plugins-bundled plugins-bundled
 
 RUN yarn install --immutable
 
 COPY tsconfig.json .eslintrc .editorconfig .browserslistrc .prettierrc.js babel.config.json .linguirc ./
-COPY public public
```

Comment:
What's the reason moving this up?

Reply:
`yarn install` fails because the package.json has a reference to `@grafana-plugins/testdata` which has not been copied yet.